### ISSUE TITLE
Better error message for GBneck models when radii are out-of-range

### ISSUE THREAD

**Issue by swails** [pandegroup/openmm#925]
See #923 for the discussion.

Currently there is no range checking on the parameters given to GBneck GB force objects.  Only by knowing the original implementation (in Amber) and how it uses an interpolation table limited in range from 1 to 2 Angstroms would you know what to look for.

It would be a lot better to give an *informative* error message rather than a cryptic one.

This issue a stub to remind me to implement this.

**Comment by swails** [comment]
I tried implementing this.  Basically my approach was to subclass `CustomGBForce` and override `addParticle` to add this error-checking.

Didn't work.  I tried following http://www.swig.org/Doc1.3/Python.html#Python_nn21 for advice about getting this mixed-C++/Python inheritance stuff to work, but it never did.  And all of the problems occurred in the SWIG-generated layer -- nothing I could fix in Python that I could tell.

I guess I'll try a wrapper class next when I get around to it.
